### ISSUE TITLE
Can't use add-on in an external project

### ISSUE THREAD

**Issue by jfilby** [openpeeps/denim#7]
I successfully build an add-on, but when I copy it to an external project, I can't get it to load. This is a Next.js project. Every technique I try encounters a different error.

Do I need to wait for the ability to build an npm from the add-on?

**Comment by georgelemon** [comment]
Maybe you need Webpack [node-loader](https://webpack.js.org/loaders/node-loader/)? Or something similar but for Babel

**Comment by jfilby** [comment]
The node-loader package gives me an error that it can't find the node module in pages/api with a generated filename. There's nextjs-node-loader which gets further, but needs an older glibc package that seems to have conflicts when I try to install older rpms.

I'll leave this issue open to include any tips I may have if/when I get it working.

**Comment by jfilby** [comment]
I got it working by:
1. Using nextjs-node-loader for the Next.js I'm using the addon in. Node-loader tried to find the addon as a shared library in a path and filename that didn't exist. Node-bindings had a bug that gave an indexOf undefined error.
2. Using Node v20. I had a glibc_2.34 not found error with v18, I'm not sure if that would've been eliminated with the next step.
3. Removing any imports of the jester package from my addon. I was importing code from my Nexus package that did this to check headers and submitted form fields.
4. I import the functions from the addon using a relative path. I had to add an @ts-ignore annotation above the import to silence a VSCode warning that the import couldn't be found.
